### ISSUE TITLE
Cannot re-order Fields within a Fieldset in Control Panel

### ISSUE THREAD

**Issue by andrewcgregory** [statamic/cms#962]
Fields within a Fieldset cannot be re-ordered in the Control Panel. Editing the .yaml file to change the order works. Fields in a Blueprint can be re-ordered within the Control Panel and I would expect the same functionality for Fieldsets.

**Comment by jackmcdade** [comment]
Fieldsets aren't meant to be used by themselves and are just a way to organize reusable fields that go into your Blueprints. We need to add some more docs and help text around them.

If you're coming from v2, Blueprints are what you want to use for everything now, and Fieldsets are essentials "partials" in the v2 sense. Does that...make sense? 😊

**Comment by jasonvarga** [comment]
I think it’s useful though since you can import a whole fieldset.

**Comment by andrewcgregory** [comment]
Yes, Jason that was my thought as well. I'll explain my thought process here... but of course it's up to you guys (or gentlemen may be more appropriate) if you want to add this functionality or not.

I created a Fieldset for an `address`. This Fieldset contains fields for `location_name`, `street_address`, `city`, etc. This Fieldset is then imported into multiple Blueprints to be used, including the global company address, global company mailing address, user address, etc. 

I understand that if I imported a field individually into the Blueprint I could then sort them, but instead I imported the full Fieldset. Then I added a new field `street_address_2` but was unable to move this above `city` without editing the .yaml file.
